addon/components/power-select-infinity/trigger.js

```diff
@@ -96,11 +100,30 @@ export default Component.extend({
         e.stopPropagation();
       }
 
+      if (e.keyCode === 8 && get(this, 'select.selected')) {
```

Comment:
probably would be good to have a constant for `8`, like ` const KEYCODE_BACKSPACE = 8;` or something